src/app/public/modules/autocomplete/autocomplete.component.ts

```diff
@@ -137,15 +173,29 @@ export class SkyAutocompleteComponent
       ? this._searchTextMinimumCharacters : 1;
   }
 
+/**
+ * Specifies an array of functions to call against each search result in order
+ * to filter the search results when using the default search function. When
+ * using the `search` property to specify a custom search function, you must
+ * manually apply filters inside that function. The function must return `true`
+ * or `false` for each result to indicate whether to display it in the dropdown list.
+ */
   @Input()
   public searchFilters: SkyAutocompleteSearchFunctionFilter[];
 
+/**
+ * Specifies the maximum number of search results to display in the dropdown list.
+ * By default, the component displays all matching results.
+ */
   @Input()
   public searchResultsLimit: number;
 
   @Input()
   public noResultsFoundText: string;
 
+/**
+ * Fires when users select items in the dropdown list.
+ */
   @Output()
   public get selectionChange(): EventEmitter<SkyAutocompleteSelectionChange> {
```

Comment:
This event is showing as "Optional." That doesn't really make sense for an event, and is probably a docs-tools bug. I filed an issue here: https://github.com/blackbaud/skyux-docs-tools/issues/132